components/pricing_modal/starter_disclaimer_cta.tsx

```diff
@@ -55,12 +55,11 @@ function StarterDisclaimerCTA() {
                 },
                 description: {
                     id: t('workspace_limits.modals.informational.description.freeLimits'),
-                    defaultMessage: '{planName} is restricted to {messages} message history, {storage} file storage, and {boards} board cards.',
```

Comment:
Wtihout the PR, this will still show up when downgrading, even after limit is removed.